Automattic/jetpack
# One-click site verification and search console integration

The primary purpose of this change is to simplify site verification by letting the user connect their Google account and automatically set the meta tag value and verify their site with Google in one step. It will also register with Search Console in order to set the `sitemap.xml` address.

This provides a foundation for further improvements, including:

* Automatically registering the site with Search Console
* Track indexing status of new posts
* Access to Speedy Indexing API

#### Changes proposed in this Pull Request:

* Add a button to verify site with Google
* If it's easy, add the site to Search Console at the same time

#### Testing instructions:

**Setup**
* Apply D17515-code
* Point Jetpack to Sandbox
* Set `public-api.wordpress.com` to your sandbox IP



**Testing**
* Go to Jetpack -> Settings -> Traffic
* Click "Verify with Google"
* You will be prompted to authorize the connection in a popup
* The UI should change to tell you that your site was verified (and added to Search Console?)


#### Proposed changelog entry for your changes:

Enable one-click site verification and sitemap.xml registration with Google
